### ISSUE TITLE
Fontstack

### ISSUE THREAD

**Issue by MoritzBru** [Simonwep/pickr#13]
Why include _Montserrat_ from Google Fonts and then use _Source Sans Pro_?
Wouldnt it be better using a native fontstack like below?
```scss
$font-family:
  -apple-system,
  BlinkMacSystemFont,
  "Segoe UI",
  "Roboto",
  "Helvetica Neue", Arial, sans-serif, !default;
```
This is taken from Bootstrap but without the Emoji stuff.
https://getbootstrap.com/docs/4.1/content/reboot/#native-font-stack

**Comment by Simonwep** [comment]
Oh, it seems like I've choosed Montserrat and then switched and forgot about the including. Thanks! 
Using the native fontstack would be definitely better, I will change it!

**Comment by Simonwep** [comment]
Okay, I've fixed it in 6faac0934e7f5ffd8e19d183a8177d278b9f80ba :)
